angular.json

```diff
@@ -31,6 +31,7 @@
               ]
             },
             "scripts": [
+              "node_modules/split.js/dist/split.min.js"
```

Comment:
Split.js is used for the splitting of the editing and preview panel. It also has a drawer inbetween, so the user can decide on the size

Reply:
Awesome tool, i like how it works. Good choice!